share/lib/common.sh

```diff
@@ -27,10 +27,10 @@ service_action() {
   service="$1"
   action="$2"
 
-  if [ -x /usr/sbin/service ]; then
-    CMD="service ${service} ${action}"
-  elif [ -x /etc/init.d/${service} ]; then
+  if [ -x /etc/init.d/${service} ]; then
```

Comment:
It may be an issue to manage rudder-agent out of systemd on some systems. We could test if rudder-agent is recognized as a systemd service, and use it directly in this case to avoid any issue.